app/assets/javascripts/textinputs.js

```diff
@@ -7,7 +7,10 @@ function initTextArea () {
   var $text = $('#organization_description');
   function resize () {
     $text.height( 'auto' );
-    $text.height( $('textarea')[0].scrollHeight );
+    var textareas = $('textarea');
+    if(textareas.length) {
+      $text.height(textareas[0].scrollHeight);
+    }
```

Comment:
When there are no textareas on screen, the original code was trying to access the first element of an empty list, and throwing an error. That stopped the rest of the JS from running in the specs.